app/assets/stylesheets/components/_theme.css.scss

```diff
@@ -139,6 +139,10 @@ md-input-group {
   }
 }
 
+.md-padding {
+  padding: 16px;
```

Comment:
Having a rule or a thousand doesn't matter. I'm not sure, but you're using this like an utilitie, wich is not good. The way it looks is that you have md-card, witch is a container for stuff, and you have another type of container and that's the reason you use this. The best way would be no to use this and move the rules to the components. Where are you using this?